ParkingApplication/Parking.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Net.Sockets;
+using System.Runtime.CompilerServices;
+using Microsoft.VisualBasic;
+
+namespace ParkingApplication;
+
+public class Parking
+{   
+    public int MaxSlots = 4;
```

Comment:
Ca sa iti fie mai usor in teste, poti pune un parametru in constructor care sa populeze maxslots, asa hotarasti dimensiunea cand creezi un obiect de Parking